src/stylesheets/components/_accordions.scss

```diff
@@ -1,16 +1,12 @@
 // Variables
 
-$accordion-border: 3px solid $color-gray-lightest;
+$accordion-border: 3px solid color('gray-5');
```

Comment:
For colors, it's always best to use the highest level of abstraction aka theme values. In this case, gray 5 is base lighter. See: https://v2.designsystem.digital.gov/style/color/theme-palette/

Reply:
3px would need to be changed to a 2.0 palette value.